src/main.ts

```diff
@@ -26,26 +26,26 @@ type StringMap = { [key: string]: string };
  * --verbose           ->   Make the operation more talkative
  */
 type CurlAdditionalOptions = {
-  compressed:         boolean,
-  compressedSsh:      boolean,
-  fail:               boolean,
-  failEarly:          boolean,
-  head:               boolean,
-  include:            boolean,
-  insecure:           boolean,
-  ipv4:               boolean,
-  ipv6:               boolean,
-  listOnly:           boolean,
-  location:           boolean,
-  locationTrusted:    boolean,
-  noKeepalive:        boolean,
-  output:             string,
-  showError:          boolean,
-  silent:             boolean,
-  ssl:                boolean,
-  sslv2:              boolean,
-  sslv3:              boolean,
-  verbose:            boolean,
```

Comment:
why would you change this types? 👀

Reply:
Thanks for taking time to open this, I'll try to have a deeper look and try it out as soon as I've some available time 🙏

Reply:
`undefined` should be allowed for no such config, and `false` will cause an runtime error.
https://github.com/albertodeago/curl-generator/blob/bf1925e01a686ea03da99279f685e7463554c6e9/src/main.ts#L125-L126

Reply:
These types of changes have been reverted and may be you can considered again later.